docs/content/preview/migrate/performance.md

```diff
@@ -70,30 +72,30 @@ By default, yb-voyager exports one table at a time. To improve data export, para
 
 ## Test results
 
-yb-voyager was tested using varying configurations, including more parallel jobs, multiple disks, and a larger cluster. The tests were run using a 40MB CSV file with 120 million rows.
+yb-voyager was tested using varying configurations, including more parallel jobs, multiple disks, and a larger cluster. The tests were run using a 28GB CSV file with 350 million rows on YugabyteDB version 2.16.0.0-b90.
 
 The table schema for the test was as follows:
 
 ```sql
-CREATE TABLE topology_flat (
-    userid_fill uuid,
-    idtype_fill text,
-    userid uuid,
-    idtype text,
-    level int,
-    locationgroupid uuid,
-    locationid uuid,
-    parentid uuid,
-    attrs jsonb,
-    PRIMARY KEY (userid, level, locationgroupid, parentid, locationid)
+CREATE TABLE public.accounts (
+    block bigint NOT NULL,discr
+    address text NOT NULL,
+    dc_balance bigint DEFAULT 0 NOT NULL,
+    dc_nonce bigint DEFAULT 0 NOT NULL,
+    security_balance bigint DEFAULT 0 NOT NULL,
+    security_nonce bigint DEFAULT 0 NOT NULL,
+    balance bigint DEFAULT 0 NOT NULL,
+    nonce bigint DEFAULT 0 NOT NULL,
+    staked_balance bigint,
+    PRIMARY KEY (block, address)
 );
 ```
 
 As more optimizations are introduced, average throughput increases. The following table shows the results.
 
 | Run | Cluster configuration | yb-voyager flags | CPU usage | Average throughput |
 | :-- | :-------------------- | :--------------- | :-------- | :----------------- |
-| 3 parallel jobs | 3 node [RF](../../architecture/docdb-replication/replication/#replication-factor) 3 cluster, c5.4x large (16 cores 32 GB) <br> 1 EBS Type gp3 disk per node, 3000 IOPS, 125 MiB bandwidth | batch-size=200k<br>parallel-jobs=3 | ~50% | 43K rows/sec |
-| Increase jobs<br>(1 per core) | 3 node RF 3 cluster, c5.4x large (16 cores 32 GB) <br> 1 EBS Type gp3 disk per node, 3000 IOPS, 125 MiB bandwidth | batch-size=200k<br>parallel-jobs=48 | ~80% | 117K rows/sec |
-| Add disks | 3 node RF 3 cluster, c5.4x large (16 cores 32GB) <br> 4 EBS Type gp3 disks per node, 3000 IOPS, 125 MiB bandwidth | | ~90% | 193K rows/sec |
-| Add nodes | 6 Node RF 3 cluster, c5.4x large (16 cores 32GB) <br> 4 EBS Type gp3 disks per node, 3000 IOPS, 125 MiB bandwidth | | ~90% | 227K rows/sec |
+| 24 parallel jobs (default) | 3 node [RF](../../architecture/docdb-replication/replication/#replication-factor) 3 cluster, c5.4x large (16 cores 32 GB) <br> 1 EBS Type gp3 disk per node, 10000 IOPS, 500 MiB bandwidth | batch-size=20k<br>parallel-jobs=24 | ~80% | 44014 rows/sec |
```

Comment:
maybe put each item in configuration column on separate line

cpu usage is all greater than 80%, yet above says don't go above 50-60?

Reply:
@ddhodge - I think it only means that we want the user to reduce the number of jobs if cpu usage goes above 50-60. I wonder if we are actually implying nit to go higher . But @priyanshi-yb , maybe you can tell us about it.

Reply:
Yes, we don't recommend to higher than 50-60% and if it goes higher we recommend to reduce the parallel jobs. These results are for test purpose only to give users a ballpark idea of what perf numbers are there with different configurations.